vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/DragAndDropSteps.java

```diff
@@ -28,38 +27,46 @@
 import org.vividus.bdd.steps.ui.web.model.Location;
 import org.vividus.bdd.steps.ui.web.validation.IBaseValidations;
 import org.vividus.selenium.IWebDriverProvider;
+import org.vividus.ui.web.action.IJavascriptActions;
 import org.vividus.ui.web.action.search.SearchAttributes;
 
 public class DragAndDropSteps
 {
-    @Inject private IBaseValidations baseValidations;
-    @Inject private IWebDriverProvider webDriverProvider;
+    private final IWebDriverProvider webDriverProvider;
+    private final IJavascriptActions javascriptActions;
+    private final IBaseValidations baseValidations;
+
+    public DragAndDropSteps(IWebDriverProvider webDriverProvider, IJavascriptActions javascriptActions,
+            IBaseValidations baseValidations)
+    {
+        this.webDriverProvider = webDriverProvider;
+        this.javascriptActions = javascriptActions;
+        this.baseValidations = baseValidations;
+    }
 
     /**
-     * Drags the <b>origin</b> element and moves it relatively to the <b>target</b> element in
+     * Drags the <b>draggable</b> element and moves it relatively to the <b>target</b> element in
      * accordance to provided <b>location</b>.
      * <br>
      * <i>Example</i>
      * <br>
-     * <code>When I drag element located `By.xpath(//div[@class='origin'])` and drop it at RIGHT_TOP of element
+     * <code>When I drag element located `By.xpath(//div[@class='draggable'])` and drop it at RIGHT_TOP of element
      * located `By.xpath(//div[@class='target'])`</code>
-     * @param origin element to be dragged
+     * If this step doesn't work, try to use step simulating drag&amp;drop
```

Comment:
does it makes sense to add @link here to step simulating drag&amp;drop?

Reply:
it will lead users to the source code

Reply:
yup, and there will be the step that end user should use instead of this one, therefore it eliminates the need to trying find it in automplete